python-sdk/tutorials/automl-with-azureml/image-pipelines/image-classification-multiclass-batch-scoring/auto-ml-image-classification-multiclass-batch-scoring.ipynb

```diff
@@ -11,7 +11,7 @@
     "# Batch Predictions for an Image Classification model trained using AutoML\n",
     "In this notebook, we go over how you can use [Azure Machine Learning pipelines](https://docs.microsoft.com/en-us/azure/machine-learning/tutorial-pipeline-batch-scoring-classification) to run a batch scoring image classification job.\n",
     "\n",
-    "**Please note:** Before running this notebook, you need to train an image classification model using AutoML for Images. Refer to the official [documentation](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models) to know how to train an image classification model with AutoML. "
+    "**Please note:** Before running this notebook, you need to train an image classification model using AutoML for Images. Refer to the official [documentation](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models) or to the [image classification multiclass notebook](https://github.com/Azure/azureml-examples/blob/main/python-sdk/tutorials/automl-with-azureml/image-classification-multiclass/auto-ml-image-classification-multiclass.ipynb) to know how to train an image classification model with AutoML. "
```

Comment:
this is not true anymore - we should add something along the lines "we include below a sample training for convenience and to be able to run the notebook end to end "

Reply:
How does this sound?

**Please note:** For this notebook you can use an existing image classification model trained using AutoML for images or use the simple model training we included below for convenience. For detailed instructions on how to train an image classification model with AutoML, please refer to the official [documentation](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models) and to the [image classification multiclass notebook](https://github.com/Azure/azureml-examples/blob/main/python-sdk/tutorials/automl-with-azureml/image-classification-multiclass/auto-ml-image-classification-multiclass.ipynb).

Reply:
Updated. Please check.